cpp/src/io/json/reader_impl.cu

```diff
@@ -345,7 +345,7 @@ std::pair<std::vector<std::string>, col_map_ptr_type> get_column_names_and_map(
   rmm::cuda_stream_view stream)
 {
   // If file only contains one row, use the file size for the row size
-  uint64_t first_row_len = d_data.size() / sizeof(char);
+  uint64_t first_row_len = d_data.size();
```

Comment:
Is this value ever used? It looks like the next 20-ish lines always overwrite whatever value is stored here. If that's the case, why not just declare it without a value? Am I missing something?

Reply:
This looks fine after all.